api/proto/teleport/legacy/types/types.proto

```diff
@@ -7042,6 +7042,9 @@ message PluginOktaSyncSettings {
   // that has not been yet updated and during the next update the value will be set to "org" if
   // AppID is empty or "saml_app" if not.
   string user_sync_source = 11;
+
+  // EnableSystemLogExport enables the Teleport Identity Security SIEM integration for Okta.
+  bool enable_system_log_export = 12;
```

Comment:
Sorry for the last minute comment. I started looking at the implementation and it looks to me it should perhaps be named `sync_system_log`? It's more of an import from Okta rather than export from what I understand.